new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/WITAssertion.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.kubernetes.assertions.impl;
+
+import oracle.weblogic.kubernetes.utils.ExecCommand;
+import oracle.weblogic.kubernetes.utils.ExecResult;
+
+import static oracle.weblogic.kubernetes.extensions.LoggedTest.logger;
+
+/**
+ * Assertions for the results of WebLogic Image Tool operations
+ */
+
+public class WITAssertion {
+
+  /**
+   * Check if a Docker image exists.
+   * @param imageName - the name of the image to be checked
+   * @param imageTag  - the tag of the image to be checked
+   * @return true if the image does exist, false otherwise
+   */
+  public static boolean imageExists(String imageName, String imageTag) {
+    logger.info("Checking if image " + imageName + ":" + imageTag + " exists.");
+    // verify the docker image is created
+    try {
+      ExecResult result = ExecCommand.exec(
+          "docker images | grep " 
+          + imageName
+          + " | grep imageTag "
```

Comment:
imageTag is a variable right? not a string literal?

Reply:
why not grep for `imageName + ":" + imageTag`

Reply:
or even better:
`docker images -q imageName:imageTag` which will return a single line with the image hash tag if it exists - no nothing if the image does not exist

Reply:
changed the code to use docker images -q imageName:imageTag.